src/Sylius/Behat/Resources/config/suites/api.yml

```diff
@@ -39,16 +39,16 @@ imports:
     - api/product/managing_product_attributes.yml
     - api/product/managing_product_options.yml
     - api/product/managing_product_reviews.yml
-#    - api/product/managing_product_variants.yml
-#    - api/product/managing_product_variants_ajax.yml
+    - api/product/managing_product_variants.yml
+    - api/product/managing_product_variants_ajax.yml
```

Comment:
These ajax endpoints should probably not be needed in 2.0 and we will want to get rid of them

Reply:
There is a task to remove them